vips/vip-234/bsctestnet.ts

```diff
@@ -0,0 +1,189 @@
+  },
+];
+
+export const vip219Testnet = () => {
+  const meta = {
+    version: "v2",
+    title: `VIP-219 VToken Upgrade`,
+    description: `If passed, this VIP will upgrade the implementation of every market in the Core pool - except vBNB (non upgradable), vLUNA and vUST (deprecated) and the markets in Isolated Pools allowing for reserves to reduced with either cash on hand or accrued revenues whichever is less.
+
+    #### Description
+
+    A situation can arise where the accrued revenues to reduce from the reserves are less than cash on hand. This can cause problems because interest accrual will throw an error that not enough cash is available and break features such as calculated fresh account balances.
+
+
+    **Audit reports**
+
+    **Deployed contracts**
+
+    - Mainnet: ****[new VToken implementations](https://bscscan.com/address/<>)
+    - Testnet: ****[new VToken implementations](https://testnet.bscscan.com/address/<>)
+
+    **References**
+
+    - [Pull request with the new Core Pool VToken contracts](https://github.com/VenusProtocol/venus-protocol/pull/414)
+    - [Pull request with the new Isolated Pools VToken contracts](https://github.com/VenusProtocol/isolated-pools/pull/337)
+    - [Simulation post upgrade](https://github.com/VenusProtocol/vips/pull/<>)
+    - [Testnet deployment Core VToken](https://testnet.bscscan.com/tx/0xaa554add9a0ab1894beb97547b855715e0ca2ab2bd7cffb59678b199867713fb)
+    - [Testnet deployment IL VToken](https://testnet.bscscan.com/tx/0x8506ab53fe5325fbaef0410e20c31cc0e384424648a6ebedc611e80dad94e0fe)
+    - [Documentation](https://docs-v4.venus.io/technical-reference/reference-technical-articles/automatic-income-allocation)`,
+    forDescription:
+      "I agree that Venus Protocol should proceed with VToken Upgrade allowing for income to be reduced based on cash on hand",
+    againstDescription:
+      "I do not think that Venus Protocol should proceed with VToken Upgrade allowing for income to be reduced based on cash on hand",
+    abstainDescription:
+      "I am indifferent to whether Venus Protocol proceeds with VToken Upgrade allowing for income to be reduced based on cash on hand",
+  };
+
+  return makeProposal(
+    [
+      ...CORE_MARKETS.map(asset => {
+        return {
+          target: asset.address,
+          signature: "_setImplementation(address,bool,bytes)",
+          params: [NEW_VBEP20_DELEGATE_IMPL, false, "0x"],
+        };
+      }),
+      // TRX is missing a set ProtocolShareReserve contract in testnet
+      {
+        target: "0x6AF3Fdb3282c5bb6926269Db10837fa8Aec67C04",
+        signature: "setProtocolShareReserve(address)",
+        params: ["0x25c7c7D6Bf710949fD7f03364E9BA19a1b3c10E3"],
+      },
+      {
+        target: VTOKEN_BEACON,
+        signature: "upgradeTo(address)",
+        params: [IL_VTOKEN_IMPL],
+      },
+      {
+        target: SN_BNB_BEACON,
+        signature: "upgradeTo(address)",
+        params: [IL_VTOKEN_IMPL],
+      },
```

Comment:
@Debugger022 , it seems the beacon contract for the vslisBNB contract is not the same one used for the rest of the vTokens. I suppose we should take that into account in the [here](vips/vip-Gateway/bsctestnet2.ts)

Reply:
Added in the Gateway VIP: https://github.com/VenusProtocol/vips/pull/195/commits/6907e771a235d151dd3e11358cb547e56ab31b6d